sdk/examples/how_tos/native_tokens/create.rs

```diff
@@ -59,13 +60,16 @@ async fn main() -> Result<()> {
         println!("Account synced");
     }
 
+    let metadata =
+        Irc30Metadata::new("My Native Token Foundry", "MNT", 10).with_description("A foundry for minting MNT NFTs.");
```

Comment:
The metadata is in the foundry output, but it's about the native token minted by it, so I think it should be rather something like this (feel free to edit the description, in the TIP the example is `FooCoin is the utility and governance token of FooLand, a revolutionary protocol in the play-to-earn crypto gaming field."`)
```suggestion
        Irc30Metadata::new("My Native Token", "MNT", 10).with_description("A native token to test the iota-sdk.");
```